test/controllers/rentals_controller_test.rb

```diff
@@ -0,0 +1,77 @@
+require "test_helper"
+
+describe RentalsController do
+  let(:customer_fixture) {customer(:customer_1)}
+  let(:video_fixture) {video(:video_1)}
+  let(:rental_fixture1){rental(:rental_1)}
+  let(:not_avail_rental_fixture){rental(:rental_2)}
+  let(:good_rental_data){
+    {
+    customer_id: customer_fixture.id,
+    video_id: video_fixture.id
+    }
+  }
+  let(:bad_rental_data1){
+    {
+    customer_id: "horse",
+    video_id: 1
+    }
+  }
+  let(:bad_rental_data2){
+    {
+    customer_id: 1,
+    video_id: "horse"
+    }
+  }
+  
+  describe "checkout" do 
+    it "good data: responds with success, created new rental" do
+      expect {
+        post check_out_path, params: good_rental_data
+      }.must_differ 'Rental.count', 1 
+      must_respond_with :created
```

Comment:
This test fails because the controller returns :success rather than :created.
It looks like y'all divided up who was doing the test and source code for this and maybe didn't run the tests after putting them together.